src/bin/dual-iir.rs

```diff
@@ -1,3 +1,30 @@
+//! # Dual IIR
+//!
+//! The Dual IIR application exposes two configurable channels. Stabilizer samples input at a fixed
+//! rate, digitally filters the data, and then generates filtered output signals on the respective
+//! channel outputs.
+//!
+//! ## Features
+//! * Two indpenendent channels
+//! * up to 800 kHz rate, timed sampling
+//! * Run-time filter configuration
+//! * Input/Output data streaming
+//! * Down to 2 µs latency
+//! * f32 IIR math
+//! * Generic biquad (second order) IIR filter
+//! * Anti-windup
+//! * Derivative kick avoidance
+//!
+//! ## Settings
+//! Refer to the [Settings] structure for documentation of run-time configurable settings for this
+//! application.
+//!
+//! ## Telemetry
+//! Refer to [Telemetry] for information about telemetry reported by this application.
```

Comment:
The lib Telemetry page is a bit thin. I would remove this section as everything has been sait in `networking.md#Telemetry`.

Reply:
The goal here was to somehow link the user to definitions for telemetry in source code in case they change in the future.

Reply:
Ok. Since they both currently just use the default Telemetry, we should pimp the `Telemetry` docs page. I think this is then only about making the struct members' docs visible.

Reply:
With the telemetry member docs now public, the structure should have more information about what data it contains, which was the original goal here.